test/e2e/main_test.go

```diff
@@ -136,125 +135,127 @@ func TestAllNS(t *testing.T) {
-	}
-
-	for name, f := range testFuncs {
-		t.Run(name, f)
-	}
-}
-
-// TestPromInstanceNs tests prometheus operator in different scenarios when --prometheus-instance-namespace is given
-func TestPromInstanceNs(t *testing.T) {
-	skipPrometheusTests(t)
-	testFuncs := map[string]func(t *testing.T){
-		"AllNs":     testPrometheusInstanceNamespaces_AllNs,
-		"AllowList": testPrometheusInstanceNamespaces_AllowList,
-		"DenyList":  testPrometheusInstanceNamespaces_DenyList,
-	}
-
-	for name, f := range testFuncs {
-		t.Run(name, f)
-	}
-}
-
-// TestAlertmanagerInstanceNs tests prometheus operator in different scenarios when --alertmanager-instance-namespace is given
-func TestAlertmanagerInstanceNs(t *testing.T) {
-	skipAlertmanagerTests(t)
-	testFuncs := map[string]func(t *testing.T){
-		"AllNs":  testAlertmanagerInstanceNamespaces_AllNs,
-		"DenyNs": testAlertmanagerInstanceNamespaces_DenyNs,
+		//"PromCreateDeleteCluster":                testPromCreateDeleteCluster,
+		//"PromScaleUpDownCluster":                 testPromScaleUpDownCluster,
+		//"PromNoServiceMonitorSelector":           testPromNoServiceMonitorSelector,
+		//"PromVersionMigration":                   testPromVersionMigration,
+		//"PromResourceUpdate":                     testPromResourceUpdate,
+		//"PromStorageLabelsAnnotations":           testPromStorageLabelsAnnotations,
+		//"PromStorageUpdate":                      testPromStorageUpdate,
+		//"PromReloadConfig":                       testPromReloadConfig,
+		//"PromAdditionalScrapeConfig":             testPromAdditionalScrapeConfig,
+		//"PromAdditionalAlertManagerConfig":       testPromAdditionalAlertManagerConfig,
+		//"PromReloadRules":                        testPromReloadRules,
+		//"PromMultiplePrometheusRulesSameNS":      testPromMultiplePrometheusRulesSameNS,
+		//"PromMultiplePrometheusRulesDifferentNS": testPromMultiplePrometheusRulesDifferentNS,
+		//"PromRulesExceedingConfigMapLimit":       testPromRulesExceedingConfigMapLimit,
+		//"PromRulesMustBeAnnotated":               testPromRulesMustBeAnnotated,
+		//"PromtestInvalidRulesAreRejected":        testInvalidRulesAreRejected,
+		//"PromOnlyUpdatedOnRelevantChanges":       testPromOnlyUpdatedOnRelevantChanges,
+		//"PromWhenDeleteCRDCleanUpViaOwnerRef":    testPromWhenDeleteCRDCleanUpViaOwnerRef,
+		//"PromDiscovery":                          testPromDiscovery,
+		//"PromAlertmanagerDiscovery":              testPromAlertmanagerDiscovery,
+		//"PromExposingWithKubernetesAPI":          testPromExposingWithKubernetesAPI,
+		//"PromDiscoverTargetPort":                 testPromDiscoverTargetPort,
+		//"PromOpMatchPromAndServMonInDiffNSs":     testPromOpMatchPromAndServMonInDiffNSs,
+		//"PromGetAuthSecret":                      testPromGetAuthSecret,
+		//"PromArbitraryFSAcc":                     testPromArbitraryFSAcc,
+		//"PromTLSConfigViaSecret":                 testPromTLSConfigViaSecret,
+		"PromRemoteWriteWithTLS": testPromRemoteWriteWithTLS,
```

Comment:
fyi these are all subtests so you can run them without uncommenting the rest with:

```
make test-e2e TEST_RUN_ARGS="-run TestAllNS/y/PromRemoteWriteWithTLS"
```

Reply:
Thanks, I didn't know that.